@xen-orchestra/vmware-explorer/esxi.mjs

```diff
@@ -357,4 +358,36 @@ export default class Esxi extends EventEmitter {
   powerOn(vmId) {
     return this.#exec('PowerOnVM_Task', { _this: vmId })
   }
+  async fetchProperty(type, id, propertyName) {
```

Comment:
Is it supposed to be used somewhere?

Reply:
yes in PR 7539 that will be reviewable in a few days